ukbb_qc/release/prepare_vcf_data_release.py

```diff
@@ -450,22 +462,26 @@ def main(args):
                 {"vep": {"Description": hl.eval(ht.vep_csq_header)}}
             )
 
-            # NOTE: Correcting rsid here because this was discovered after release HT was created
+            # Re-add rsid to HT and reformat
```

Comment:
Also make a note about fixing this on the release HT if/when we release it as well.

Reply:
Specify in the comment, why it needs to be re-added